viewer/packages/cad-geometry-loaders/src/CadManager.ts

```diff
@@ -83,6 +83,8 @@ export class CadManager {
         sector.levelOfDetail === LevelOfDetail.Detailed
       ) {
         cadModel.batchGeometry(sector.geometryBatchingQueue, sector.metadata.id);
+      } else if (sector.levelOfDetail === LevelOfDetail.Discarded) {
```

Comment:
Ideally, this should fully revert the changes made in `viewer/packages/cad-geometry-loaders/src/CadManager.ts` in #2455 . That PR removed the following from this file: 

```     
      if (sector.instancedMeshes && sector.levelOfDetail === LevelOfDetail.Detailed) {
        cadModel.updateInstancedMeshes(sector.instancedMeshes, sector.modelIdentifier, sector.metadata.id);
      } else if (sector.levelOfDetail === LevelOfDetail.Simple || sector.levelOfDetail === LevelOfDetail.Discarded) {
        cadModel.discardInstancedMeshes(sector.metadata.id);
        cadModel.removeBatchedSectorGeometries(sector.metadata.id);
      }
      ```
 Is the rest of the changes irrelevant, or should we add more back into this file?